README.md

```diff
@@ -109,22 +109,31 @@ Response
 
 ## Control API
 
-### `POST /control/fetch_blocks`
+### `POST /control/block`
```

Comment:
May provoke misunderstanding (like stop). Maybe `setBlock` or `fetchFrom` or `SetStartingBlock`?

Reply:
`setBlock` is indeed better, or even `setStartingBlock`